src/main/java/lotto/domain/LottoPapers.java

```diff
@@ -25,21 +25,20 @@ private Integer paperCount(Integer inputMoney) {
         return inputMoney / LOTTO_PRICE;
     }
 
-    private List<Integer> getAutoLottoNums() {
+    private List<LottoNum> getAutoLottoNums() {
         return LottoNumberUtil.getAutoLottoNums();
     }
 
     public List<LottoPaper> getPapers() {
-        return papers;
+        return Collections.unmodifiableList(papers);
     }
 
-    public Integer getPapersCount() {
-        return papers.size();
-    }
-
-    public String getPaperNumbersToString() {
-        return papers.stream()
-                .map(LottoPaper::getLottoNumbers).map(String::valueOf)
-                .collect(Collectors.joining(LOTTO_PAPER_JOINING_DELIMITER));
+    public WinningTable makeWinningTable(List<Integer> winningNumbers) {
+        WinningTable winningTable = new WinningTable();
+        papers.stream()
+                .forEach(lottoPaper -> {
+                    winningTable.setAutoIncrementMatchCountResult(lottoPaper.getMatchCount(winningNumbers));
+        });
```

Comment:
다음 링크를 참고하시면 좋을 것 같습니다.
https://woowacourse.github.io/javable/2020-05-14/foreach-vs-forloop

```suggestion
       papers.forEach(lottoPaper -> 
                winningTable.setAutoIncrementMatchCountResult(lottoPaper.getMatchCount(winningNumbers)));
```